src/ISA/ISA/ARCTypes/ARCTable.fs

```diff
@@ -1,12 +1,69 @@
 ﻿namespace ISA
 
+
 type ArcTable = 
     {
         Name : string
         ValueHeaders : CompositeHeader []
+        /// Key: Column * Row
         Values : System.Collections.Generic.Dictionary<int*int,CompositeCell>  
     }
 
+    /// Create ArcTable with empty 'ValueHeader' and 'Values' 
+    static member init(name: string) = {
+        Name = name
+        ValueHeaders = [||]
+        Values = System.Collections.Generic.Dictionary<int*int,CompositeCell>()
+    }
+
+    member this.ColumnCount = this.ValueHeaders.Length
+    member this.RowCount = this.Values.Keys |> Seq.maxBy snd |> snd
+
+    member this.addColumn (header:CompositeHeader, ?cells: CompositeCell [], ?index: int) = 
+        let cells = Option.defaultValue [||] cells
+        let column = Array.singleton <| CompositeColumn.create(header, cells)
+        this.addColumns (column, ?index = index)
+
+    member this.addColumns ( columns: CompositeColumn [], ?index: int) = 
+        let index = Option.defaultValue this.ColumnCount index
+        columns |> Array.iter (fun x -> x.validate(true) |> ignore)
+        let nNewColumn = columns.Length
+        let columnHeaders, columnRows = columns |> Array.map (fun x -> x.Header, x.Cells) |> Array.unzip
+        let nextHeaders = 
+            let ra = ResizeArray(this.ValueHeaders)
+            ra.InsertRange(index, columnHeaders)
+            ra.ToArray()
+        let maxRows = 
```

Comment:
Add to and use static properties of the table instead?

Reply:
Just noticed the "Add to" was already done above

Reply:
To my surprise it does not allow using both curried and tupled arguments between static members and members:

![image](https://github.com/nfdi4plants/ISADotNet/assets/39732517/c39fa994-bb67-4ccd-956c-3d3358012f91)

So this is actually a decision we need to make. static members without optional parameters or no f# piping.

This also relates to: https://github.com/nfdi4plants/ISADotNet/pull/110#discussion_r1228289834

Reply:
Even for different capitalization?

Reply:
🤕 wow, i am sorry

![image](https://github.com/nfdi4plants/ISADotNet/assets/39732517/c58f5ea6-400e-415f-8200-fa579a1e2d81)